frontend/src/app/homepage-offer/homepage-offer.component.ts

```diff
@@ -1,8 +1,9 @@
 import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
 
-import { Offer } from './offers.model';
+import { ApiOffer } from './offers.model';
 import { OffersService } from './offers.service';
 import { OrganizationService } from '../organization/organization.service';
+import { OnChanges } from '@angular/core';
```

Comment:
We don't need this line, but I think I know what you wanted to do. The offer list does not refresh, does it? ;) See my next comment in this file on how to solve this.

Reply:
Please delete this import, it is not needed